credential/exchange/model.go

```diff
@@ -122,6 +122,13 @@ type ClaimFormat struct {
 	LDPVP *LDPType `json:"ldp_vp,omitempty" validate:"omitempty,dive"`
 }
 
+func SupportedClaimFormats() map[string]bool {
+	supportedFormats := map[string]bool{
+		string(JWT): true, string(JWTVC): true, string(JWTVP): true, string(LDP): true, string(LDPVC): true, string(LDPVP): true,
```

Comment:
nit: consider a utility method on the formats so you can do JWT.String() or similar